src/versionbits.cpp

```diff
@@ -49,20 +57,17 @@ ThresholdState AbstractThresholdConditionChecker::GetStateFor(const CBlockIndex*
         pindexPrev = vToCompute.back();
         vToCompute.pop_back();
 
+        // We track state by previous-block, so the height we should be comparing is +1
+        const int64_t height = pindexPrev->nHeight + 1;
```

Comment:
Why are you widening height from an `int` (32 bit) to a `int64_t`, and then comparing with `int`s below?

Reply:
Will fix I need to retouch.